patch.go

```diff
@@ -43,10 +42,7 @@ func newLazyNode(raw *json.RawMessage) *lazyNode {
 func (n *lazyNode) MarshalJSON() ([]byte, error) {
 	switch n.which {
 	case eRaw:
-		if n.raw == nil {
-			return nil, errors.New("Missing value")
-		}
-		return *n.raw, nil
+		return json.Marshal(n.raw)
```

Comment:
Yes-- this was broken in the upstream repository, and fixed a year ago after we forked:

https://github.com/evanphx/json-patch/commit/0b3b8afc8488c222a8cd99b80fccc2ba0b01ecc9

Prior to that fix, the library would panic upon receiving an attempt to set to `null`. @drichelson did a systemic pass attempting to eliminate panics, and applied the wrong fix in this case.